### PR TITLE
ES strings 1.6 revision

### PR DESCRIPTION
**Pull request by Kiko0923**
Translation of some new strings for next release

### CODE REVIEW

File: strings/es.lproj/Vvt.strings [hunk 1]
```diff
@@ -29,10 +29,10 @@
 "%1$@ was %2$@ %3$@" = "%1$@ estaba a %2$@ %3$@";
 
 /* Notification, %1$@ is a point of interest, %2$@ is time ago, %3$@ is distance with units, '<Space Needle>, announced <1 minute ago>, <20 meters> away' */
-"%1$@, announced %2$@, %3$@ away." = "%1$@, anuncio de %2$@, %3$@ de distancia.";
+"%1$@, announced %2$@, %3$@ away." = "%1$@, aviso  de %2$@, %3$@ de distancia.";
```
Comment (drwjf) [inline strings/es.lproj/Vvt.strings:32]: Is it an extra space between `aviso` and `de`?

File: strings/es.lproj/Vvt.strings [hunk 2]
```diff
@@ -1547,10 +1547,10 @@
...
+"VoiceVista can calculate waypoints for reference based on the starting and destination locations." = "VoiceVista puede calcular puntos de referencia en función de las ubicaciones de inicio y destino.";
 
 /* No comment provided by engineer. */
-"VoiceVista can generate a new reverse route with waypoints in reverse order for your convenience. You may need to adjust the waypoints for the reversed route before starting guidance." = "VoiceVista puede generar una nueva ruta inversa con puntos de referencia en orden inverso para su comodidad. Es posible que tengas que ajustar los puntos de referencia de la ruta invertida antes de iniciar la guía.";
+"VoiceVista can generate a new reverse route with waypoints in reverse order for your convenience. You may need to adjust the waypoints for the reversed route before starting guidance." = "VoiceVista puede generar una nueva ruta inversa con puntos de referencia en orden inverso para tu  comodidad. Es posible que tengas que ajustar los puntos de referencia de la ruta invertida antes de iniciar la guía.";
```
Comment (drwjf) [inline strings/es.lproj/Vvt.strings:1553]: Is it an extra space between `tu` and `comodidad`?

**Review by drwjf — APPROVED** [review]

**Comment by Kiko0923** [comment]
NO. Extra spaces are my fault. Just remove them please.

**Comment by Kiko0923** [comment]
The same as before. Extra spaces are my fault.

Best

**Pull request merged**